apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -535,11 +535,19 @@ export const notifications = {
       description:
         'Fyrirsögn í pósti til verjanda þegar hann er skráður á mál.',
     },
+    // TODO: REMOVE body
     body: {
       id: 'judicial.system.backend:notifications.defender_assigned_email.body',
       defaultMessage:
         '{court} hefur skipað þig verjanda í máli {linkStart}{courtCaseNumber}{linkEnd}. Gögn málsins eru aðgengileg í Réttarvörslugátt með rafrænum skilríkjum.',
       description: 'Texti í pósti til verjanda þegar hann er skráður á mál.',
     },
+    bodyV2: {
+      id:
+        'judicial.system.backend:notifications.defender_assigned_email.body_v2',
```

Comment:
Where is this being used? I don´t see it in this diff.

Reply:
It's here 👉  https://github.com/island-is/island.is/pull/8939/files#diff-b2788662cb5317437f7ba480f353c0d321e3181497eb8d1accfed00b066ca25eR663